src/consensus/pbft/libbyz/test/replica_unit_tests.cpp

```diff
@@ -203,6 +205,7 @@ struct PbftState
   pbft::RequestsMap& pbft_requests_map;
   ccf::Signatures& signatures;
   pbft::PrePreparesMap& pbft_pre_prepares_map;
+  pbft::NewViewsMap& pbft_new_views_map;
```

Comment:
Curious: why `NewViewsMap` and not `ViewsMap`?

Reply:
It is to indicate that is is a `New_view` message that is getting stored here. I think it just makes it less ambiguous since there are more than one message types that participate in view changes